tools/bump_version.sh

```diff
@@ -37,9 +37,9 @@ update_dep_version() {
 update_image_versions() {
```

Comment:
To make the script more robust, I think that we should avoid calling `update_image_version` for workflow file that do not contain Docker images, e.g., `push_git_tag.yml`.

```
    # Update Docker image versions in workflows
    WORKFLOWS=$(find "${ASTER_SRC_DIR}/.github/workflows/" -type f -name "*.yml")
    for workflow in $WORKFLOWS; do
        update_image_versions $workflow
    done
```

The `WORKFLOWS` variable should exclude workflow files that do not contain Docker image versions.